integration-tests/chopsticks/src/helper/utils.ts

```diff
@@ -0,0 +1,82 @@
+import { Keyring } from '@polkadot/keyring'
+import { u8aToHex } from '@polkadot/util'
+import { decodeAddress } from '@polkadot/util-crypto'
+import { ExpectStatic } from 'vitest'
+
+/**
+ * Keyring instance to generate key pairs with 38 ss58Format encoding.
+ */
+const keyring = new Keyring({ type: 'ed25519', ss58Format: 38 })
+
+/**
+ * Key pairs for Alice, Bob, and Charlie used for testing.
+ */
+export const keysAlice = keyring.addFromUri('//alice', undefined, 'ed25519')
```

Comment:
Key type is redundant if the keyring already contains that info, as is the case here.